js/renovation/ui/scroll_view/bottomPocket.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  JSXComponent,
+  OneWay,
+  ComponentBindings,
+  Component,
+  Ref,
+  RefObject,
+} from 'devextreme-generator/component_declaration/common';
+
+import {
+  SCROLLVIEW_BOTTOM_POCKET_CLASS,
+} from './scrollable_utils';
+
+const SCROLLVIEW_REACHBOTTOM_CLASS = 'dx-scrollview-scrollbottom';
+const SCROLLVIEW_REACHBOTTOM_INDICATOR_CLASS = 'dx-scrollview-scrollbottom-indicator';
+const SCROLLVIEW_REACHBOTTOM_TEXT_CLASS = 'dx-scrollview-scrollbottom-text';
+
+export const viewFunction = (viewModel: BottomPocket): JSX.Element => {
+  const {
+    bottomPocketRef,
+  } = viewModel;
+
+  return (
+    <div className={SCROLLVIEW_BOTTOM_POCKET_CLASS} ref={bottomPocketRef}>
+      <div className={SCROLLVIEW_REACHBOTTOM_CLASS}>
+        <div className={SCROLLVIEW_REACHBOTTOM_INDICATOR_CLASS} />
+        <div className={SCROLLVIEW_REACHBOTTOM_TEXT_CLASS}>Updating...</div>
+      </div>
+    </div>
+  );
+};
+
+@ComponentBindings()
+export class BottomPocketProps {
+  @OneWay() someProp = false;
+}
+
+@Component({
+  view: viewFunction,
+})
+
+export class BottomPocket extends JSXComponent(BottomPocketProps) {
+  @Ref() bottomPocketRef!: RefObject<HTMLDivElement>;
```

Comment:
Is it used? Let's add it later in a PR with the code that uses it
I think we can delete the 
`const $bottomPocket = this._$bottomPocket = $('<div>').addClass(SCROLLVIEW_BOTTOM_POCKET_CLASS);`
line in this PR

Reply:
Fix in 7d6265de52141a5f6ac3292fd8eca177347407e8